environment-setup/ansible/TP2/group_vars/rancher.yml

```diff
@@ -0,0 +1,24 @@
+dns_resolver_extra: "192.168.1.1"
+k3s_admin_user: "root"
+k3s_ssh_key: "k3s_admin_key"
+k3s_ssh_key_path: "~/.ssh/"
+rancher_server_name: "rancher"
+argocd_server_name: "argocd"
+current_machine_ipaddress: "{{ hostvars['cloud'].ansible_default_ipv4.address
+  | default(hostvars['cloud'].ansible_all_ipv4_addresses[0]) }}"
+
+pv_basename: project2-pv2
```

Comment:
Can we make this `project2-pv`? (I think this last `2` may have been added due to borking the configuration of just `-pv`! :)